model/src/main/java/com/chiralbehaviors/CoRE/attribute/Attribute.java

```diff
@@ -77,8 +77,8 @@
                @NamedQuery(name = FIND_BY_NAME, query = "select e from Attribute e where e.name = :name"),
                @NamedQuery(name = GET_CHILD, query = "SELECT rn.child "
                                                      + "FROM AttributeNetwork rn "
-                                                     + "WHERE rn.parent = :parent "
```

Comment:
I think... I *think* this is github being stupid. I've checked locally, and the changes are correct there, and there are no diffs between github and my repo